operator/operator_test.go

```diff
@@ -164,7 +170,79 @@ func initEnvironment(t *testing.T) *environment {
 
 	log.Info("operator_test.initEnvironment: Created users")
 
-	return &environment{t, cfg, cmd, operator, user1, user2, errg, params}
+	return &environment{t, cfg, cmd, operator, user1, user2, errg, params, cleanup}
+}
+
+type testSetup interface {
+	setup(cfg *Config) (operator *Operator, cleanup func() error)
+}
+
+type prototypeSetup struct{}
+
+func (prototypeSetup) setup(cfg *Config) (*Operator, func() error) {
+	return SetupWithPrototypeEnclave(cfg), func() error { return nil }
```

Comment:
why don't you `Stop` the prototype enclave in the cleanup function here?

Reply:
code no longer exists